packs/classfeatures/the-distant-grasp.json

```
@@ -24,7 +24,125 @@
             "remaster": false,
             "title": "Pathfinder Dark Archive"
         },
-        "rules": [],
+        "rules": [
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "or": [
```

```suggestion
                        "spellcasting:category:spontaneous",
                        "spellcasting:tradition:occult",
                        "or": [
```
Best we can do for now to not step on innate/archetype spell collections.